blocks/results-list-block/features/results-list/results/result-item.jsx

```diff
@@ -1,86 +1,151 @@
 import React from "react";
+import { useEditableContent } from "fusion:content";
+import { useComponentContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import { RESIZER_APP_VERSION, RESIZER_URL } from "fusion:environment";
+import { localizeDateTime } from "@wpmedia/engine-theme-sdk";
 
-import ArticleDate from "@wpmedia/date-block";
-import { Image } from "@wpmedia/engine-theme-sdk";
-import { extractResizedParams, extractImageFromStory } from "@wpmedia/resizer-image-block";
-import { Byline, Heading, SecondaryFont, Overline } from "@wpmedia/shared-styles";
+import {
+	Attribution,
+	Separator,
+	Heading,
+	Overline,
+	Image,
+	Link,
+	Date as DateComponent,
+	formatAuthors,
+	Paragraph,
+	Join,
+	formatURL,
+	MediaItem,
+	Conditional,
+	getImageFromANS,
+	imageANSToImageSrc,
+	usePhrases,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-results-list";
 
 const ResultItem = React.memo(
 	React.forwardRef(
 		(
 			{
 				arcSite,
 				element,
-				imageProperties,
 				targetFallbackImage,
-				placeholderResizedImageOptions,
 				showByline,
 				showDate,
 				showDescription,
 				showHeadline,
 				showImage,
 				showItemOverline,
+				overlineURL,
 			},
 			ref
 		) => {
 			const {
 				description: { basic: descriptionText } = {},
 				display_date: displayDate,
+				label: { basic = {} } = {},
 				headlines: { basic: headlineText } = {},
 				websites,
+				credits,
 			} = element;
-
-			const imageURL = extractImageFromStory(element);
+			const {
+				dateLocalization: { language, timeZone, dateTimeFormat } = {
+					language: "en",
+					timeZone: "GMT",
+					dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+				},
+			} = getProperties(arcSite);
+			const overline = basic?.text || "Overline";
```

Comment:
We should not be default the `overline` to be an English string of text, overline should be derived from ANS in the first instance. Also this is missing the sponsored content logic from the Overline component that was previously used from Shared styles